src/drive/web/modules/upload/index.js

```diff
@@ -138,30 +176,55 @@ const uploadFile = async (client, file, dirID) => {
   return resp.data
 }
 
+export const getFilePath = async (client, file, dirID) => {
```

Comment:
Should we rename to something more accurate? Does `parentDirectory.path` return the fullpath / absolute path of the folder?

Reply:
Yes, `parentDirectory.path == '/Administratif/Whatever`
Something more accurate like `getFileFullpath` ?

Reply:
yep

Reply:
yep